common/openapi.yml

```diff
@@ -265,6 +317,77 @@ components:
           items:
             $ref: '#/components/schemas/WorkflowOutputDefinition'
           description: Declarative output definition for writing back to WDS.
+    RunStateRequest:
+      type: object
+      required: [runId, state]
+      properties:
+        runId:
+          type: string
+          format: uuid
+        state:
+          $ref: '#/components/schemas/RunState'
+        workflow_output_definitions:
```

Comment:
@jgainerdewar: I think I got this wrong and instead of output definition, the body should have wds_records here coming from cromwell. Please confirm - I'll update the request bodies.

Reply:
Agreed, I think it makes sense to use a new scheme.